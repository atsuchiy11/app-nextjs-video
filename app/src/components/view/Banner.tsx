import * as React from 'react'
import { makeStyles, Theme } from '@material-ui/core/styles'

import Card from '@material-ui/core/Card'
import CardActionArea from '@material-ui/core/CardActionArea'
import CardMedia from '@material-ui/core/CardMedia'
import Typography from '@material-ui/core/Typography'
import Box from '@material-ui/core/Box'

const useStyles = makeStyles((theme: Theme) => ({
	wraper: {
		'& .MuiBox-root::-webkit-scrollbar': {
			display: 'none',
		},
	},
	root: {
		overflowX: 'scroll',
		whiteSpace: 'nowrap',
		marginTop: theme.spacing(1),
		marginBottom: theme.spacing(1),
	},
	card: {
		width: '40%',
		display: 'inline-block',
		marginRight: theme.spacing(1),
	},
	media: {
		height: 0,
		paddingTop: '56.25%', //16:9,
		position: 'relative',
	},
	description: {
		position: 'absolute',
		bottom: 0,
		right: 0,
		backgroundColor: theme.palette.primary.dark,
		opacity: 0.75,
		width: '100%',
	},
	overflow: {
		overflowX: 'hidden',
		whiteSpace: 'nowrap',
		textOverflow: 'ellipsis',
	},
}))

const Banner: React.FC = () => {
	const classes = useStyles()

	return (
		<div className={classes.wraper}>
			<Box className={classes.root}>
				{[0, 1, 2, 3, 4, 5].map((index) => (
					<Card
						className={classes.card}
						key={index}
						data-testid="banner">
						<CardActionArea>
							<CardMedia
								className={classes.media}
								image="/design.jpg"
								title="Banner Image"
								data-testid="banner-image">
								<Box className={classes.description}>
									<Typography
										variant="subtitle2"
										data-testid="banner-title">
										title
									</Typography>
									<Typography
										variant="caption"
										component="p"
										className={classes.overflow}
										data-testid="banner-text">
										Learning path description ... Lorem
										ipsum dolor sit amet, consectetur
										adipisicing elit, sed do eiusmod tempor
										incididunt ut labore et dolore magna
										aliqua. Ut enim ad minim veniam, quis
										nostrud exercitation ullamco laboris
										nisi ut aliquip ex ea commodo consequat.
										Duis aute irure dolor in reprehenderit
										in voluptate velit esse cillum dolore eu
										fugiat nulla pariatur. Excepteur sint
										occaecat cupidatat non proident, sunt in
										culpa qui officia deserunt mollit anim
										id est laborum.
									</Typography>
								</Box>
							</CardMedia>
						</CardActionArea>
					</Card>
				))}
			</Box>
		</div>
	)
}
export default Banner
