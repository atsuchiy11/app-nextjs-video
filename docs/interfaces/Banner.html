<!DOCTYPE html><html class="default no-js"><head><meta charSet="utf-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><title>Banner | prime-studio</title><meta name="description" content="Documentation for prime-studio"/><meta name="viewport" content="width=device-width, initial-scale=1"/><link rel="stylesheet" href="../assets/style.css"/><link rel="stylesheet" href="../assets/highlight.css"/><script async src="../assets/search.js" id="search-script"></script></head><body><header><div class="tsd-page-toolbar"><div class="container"><div class="table-wrap"><div class="table-cell" id="tsd-search" data-base=".."><div class="field"><label for="tsd-search-field" class="tsd-widget search no-caption">Search</label><input type="text" id="tsd-search-field"/></div><ul class="results"><li class="state loading">Preparing search index...</li><li class="state failure">The search index is not available</li></ul><a href="../index.html" class="title">prime-studio</a></div><div class="table-cell" id="tsd-widgets"><div id="tsd-filter"><a href="#" class="tsd-widget options no-caption" data-toggle="options">Options</a><div class="tsd-filter-group"><div class="tsd-select" id="tsd-filter-visibility"><span class="tsd-select-label">All</span><ul class="tsd-select-list"><li data-value="public">Public</li><li data-value="protected">Public/Protected</li><li data-value="private" class="selected">All</li></ul></div> <input type="checkbox" id="tsd-filter-inherited" checked/><label class="tsd-widget" for="tsd-filter-inherited">Inherited</label><input type="checkbox" id="tsd-filter-externals" checked/><label class="tsd-widget" for="tsd-filter-externals">Externals</label></div></div><a href="#" class="tsd-widget menu no-caption" data-toggle="menu">Menu</a></div></div></div></div><div class="tsd-page-title"><div class="container"><ul class="tsd-breadcrumb"><li><a href="../modules.html">prime-studio</a></li><li><a href="Banner.html">Banner</a></li></ul><h1>Interface Banner</h1></div></div></header><div class="container container-main"><div class="row"><div class="col-8 col-content"><section class="tsd-panel tsd-comment"><div class="tsd-comment tsd-typography"><div class="lead">
<p>Banner response body</p>
</div><dl class="tsd-comment-tags"><dt>export</dt><dd></dd><dt>interface</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-hierarchy"><h3>Hierarchy</h3><ul class="tsd-hierarchy"><li><span class="target">Banner</span></li></ul></section><section class="tsd-panel-group tsd-index-group"><h2>Index</h2><section class="tsd-panel tsd-index-panel"><div class="tsd-index-content"><section class="tsd-index-section "><h3>Properties</h3><ul class="tsd-index-list"><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#PK" class="tsd-kind-icon">PK</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#SK" class="tsd-kind-icon">SK</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#createdAt" class="tsd-kind-icon">created<wbr/>At</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#createdUser" class="tsd-kind-icon">created<wbr/>User</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#description" class="tsd-kind-icon">description</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#id" class="tsd-kind-icon">id</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#image" class="tsd-kind-icon">image</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#indexKey" class="tsd-kind-icon">index<wbr/>Key</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#invalid" class="tsd-kind-icon">invalid</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#link" class="tsd-kind-icon">link</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#name" class="tsd-kind-icon">name</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#note" class="tsd-kind-icon">note</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#updatedAt" class="tsd-kind-icon">updated<wbr/>At</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#updatedUser" class="tsd-kind-icon">updated<wbr/>User</a></li></ul></section></div></section></section><section class="tsd-panel-group tsd-member-group "><h2>Properties</h2><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="PK" class="tsd-anchor"></a><h3>PK</h3><div class="tsd-signature tsd-kind-icon">PK<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L53">api.ts:53</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="SK" class="tsd-anchor"></a><h3>SK</h3><div class="tsd-signature tsd-kind-icon">SK<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L59">api.ts:59</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="createdAt" class="tsd-anchor"></a><h3>created<wbr/>At</h3><div class="tsd-signature tsd-kind-icon">created<wbr/>At<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L65">api.ts:65</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="createdUser" class="tsd-anchor"></a><h3>created<wbr/>User</h3><div class="tsd-signature tsd-kind-icon">created<wbr/>User<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L71">api.ts:71</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="description" class="tsd-anchor"></a><h3>description</h3><div class="tsd-signature tsd-kind-icon">description<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L107">api.ts:107</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="id" class="tsd-anchor"></a><h3>id</h3><div class="tsd-signature tsd-kind-icon">id<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">number</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L41">api.ts:41</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="image" class="tsd-anchor"></a><h3><span class="tsd-flag ts-flagOptional">Optional</span> image</h3><div class="tsd-signature tsd-kind-icon">image<span class="tsd-signature-symbol">?:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L113">api.ts:113</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="indexKey" class="tsd-anchor"></a><h3>index<wbr/>Key</h3><div class="tsd-signature tsd-kind-icon">index<wbr/>Key<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L47">api.ts:47</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="invalid" class="tsd-anchor"></a><h3>invalid</h3><div class="tsd-signature tsd-kind-icon">invalid<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">boolean</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L89">api.ts:89</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="link" class="tsd-anchor"></a><h3><span class="tsd-flag ts-flagOptional">Optional</span> link</h3><div class="tsd-signature tsd-kind-icon">link<span class="tsd-signature-symbol">?:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L119">api.ts:119</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="name" class="tsd-anchor"></a><h3>name</h3><div class="tsd-signature tsd-kind-icon">name<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L101">api.ts:101</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="note" class="tsd-anchor"></a><h3><span class="tsd-flag ts-flagOptional">Optional</span> note</h3><div class="tsd-signature tsd-kind-icon">note<span class="tsd-signature-symbol">?:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L95">api.ts:95</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="updatedAt" class="tsd-anchor"></a><h3>updated<wbr/>At</h3><div class="tsd-signature tsd-kind-icon">updated<wbr/>At<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L77">api.ts:77</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section><section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface"><a name="updatedUser" class="tsd-anchor"></a><h3>updated<wbr/>User</h3><div class="tsd-signature tsd-kind-icon">updated<wbr/>User<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div><aside class="tsd-sources"><ul><li>Defined in <a href="https://github.com/prime-x-co-ltd/prime-stream/blob/19b5a23/app/src/interfaces/api/api.ts#L83">api.ts:83</a></li></ul></aside><div class="tsd-comment tsd-typography"><dl class="tsd-comment-tags"><dt>memberof</dt><dd><p>Banner</p>
</dd></dl></div></section></section></div><div class="col-4 col-menu menu-sticky-wrap menu-highlight"><nav class="tsd-navigation primary"><ul><li class=""><a href="../modules.html">Exports</a></li></ul></nav><nav class="tsd-navigation secondary menu-sticky"><ul><li class="current tsd-kind-interface"><a href="Banner.html" class="tsd-kind-icon">Banner</a><ul><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#PK" class="tsd-kind-icon">PK</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#SK" class="tsd-kind-icon">SK</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#createdAt" class="tsd-kind-icon">created<wbr/>At</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#createdUser" class="tsd-kind-icon">created<wbr/>User</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#description" class="tsd-kind-icon">description</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#id" class="tsd-kind-icon">id</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#image" class="tsd-kind-icon">image</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#indexKey" class="tsd-kind-icon">index<wbr/>Key</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#invalid" class="tsd-kind-icon">invalid</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#link" class="tsd-kind-icon">link</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#name" class="tsd-kind-icon">name</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#note" class="tsd-kind-icon">note</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#updatedAt" class="tsd-kind-icon">updated<wbr/>At</a></li><li class="tsd-kind-property tsd-parent-kind-interface"><a href="Banner.html#updatedUser" class="tsd-kind-icon">updated<wbr/>User</a></li></ul></li></ul></nav></div></div></div><footer class="with-border-bottom"><div class="container"><h2>Legend</h2><div class="tsd-legend-group"><ul class="tsd-legend"><li class="tsd-kind-property tsd-parent-kind-interface"><span class="tsd-kind-icon">Property</span></li></ul><ul class="tsd-legend"><li class="tsd-kind-method tsd-parent-kind-class"><span class="tsd-kind-icon">Method</span></li></ul><ul class="tsd-legend"><li class="tsd-kind-constructor tsd-parent-kind-class tsd-is-inherited"><span class="tsd-kind-icon">Inherited constructor</span></li></ul></div><h2>Settings</h2><p>Theme <select id="theme"><option value="os">OS</option><option value="light">Light</option><option value="dark">Dark</option></select></p></div></footer><div class="container tsd-generator"><p>Generated using <a href="https://typedoc.org/" target="_blank">TypeDoc</a></p></div><div class="overlay"></div><script src="../assets/main.js"></script></body></html>