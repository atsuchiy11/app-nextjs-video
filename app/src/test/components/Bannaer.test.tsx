import * as React from 'react'
import { cleanup, render } from '../testUtils'
import Banner from 'src/components/view/Banner'

describe('Uniit: Banner', () => {
	afterEach(cleanup)

	it('matches snapshot', () => {
		const { asFragment } = render(<Banner />, {})
		expect(asFragment()).toMatchSnapshot()
	})

	it('show banner', () => {
		const { getAllByTestId } = render(<Banner />, {})
		const preCount = 6
		expect(getAllByTestId('banner')).toHaveLength(preCount)
		expect(getAllByTestId('banner-image')).toHaveLength(preCount)
		expect(getAllByTestId('banner-title')).toHaveLength(preCount)
		expect(getAllByTestId('banner-text')).toHaveLength(preCount)
	})
})
