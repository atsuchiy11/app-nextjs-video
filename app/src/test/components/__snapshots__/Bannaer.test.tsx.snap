// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Uniit: Banner matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-wraper-1"
  >
    <div
      class="MuiBox-root MuiBox-root-7 makeStyles-root-2"
    >
      <div
        class="MuiPaper-root MuiCard-root makeStyles-card-3 MuiPaper-elevation1 MuiPaper-rounded"
        data-testid="banner"
      >
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardMedia-root makeStyles-media-4"
            data-testid="banner-image"
            style="background-image: url(/design.jpg);"
            title="Banner Image"
          >
            <div
              class="MuiBox-root MuiBox-root-8 makeStyles-description-5"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle2"
                data-testid="banner-title"
              >
                title
              </h6>
              <p
                class="MuiTypography-root makeStyles-overflow-6 MuiTypography-caption"
                data-testid="banner-text"
              >
                Learning path description ... Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
              </p>
            </div>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-card-3 MuiPaper-elevation1 MuiPaper-rounded"
        data-testid="banner"
      >
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardMedia-root makeStyles-media-4"
            data-testid="banner-image"
            style="background-image: url(/design.jpg);"
            title="Banner Image"
          >
            <div
              class="MuiBox-root MuiBox-root-9 makeStyles-description-5"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle2"
                data-testid="banner-title"
              >
                title
              </h6>
              <p
                class="MuiTypography-root makeStyles-overflow-6 MuiTypography-caption"
                data-testid="banner-text"
              >
                Learning path description ... Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
              </p>
            </div>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-card-3 MuiPaper-elevation1 MuiPaper-rounded"
        data-testid="banner"
      >
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardMedia-root makeStyles-media-4"
            data-testid="banner-image"
            style="background-image: url(/design.jpg);"
            title="Banner Image"
          >
            <div
              class="MuiBox-root MuiBox-root-10 makeStyles-description-5"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle2"
                data-testid="banner-title"
              >
                title
              </h6>
              <p
                class="MuiTypography-root makeStyles-overflow-6 MuiTypography-caption"
                data-testid="banner-text"
              >
                Learning path description ... Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
              </p>
            </div>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-card-3 MuiPaper-elevation1 MuiPaper-rounded"
        data-testid="banner"
      >
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardMedia-root makeStyles-media-4"
            data-testid="banner-image"
            style="background-image: url(/design.jpg);"
            title="Banner Image"
          >
            <div
              class="MuiBox-root MuiBox-root-11 makeStyles-description-5"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle2"
                data-testid="banner-title"
              >
                title
              </h6>
              <p
                class="MuiTypography-root makeStyles-overflow-6 MuiTypography-caption"
                data-testid="banner-text"
              >
                Learning path description ... Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
              </p>
            </div>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-card-3 MuiPaper-elevation1 MuiPaper-rounded"
        data-testid="banner"
      >
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardMedia-root makeStyles-media-4"
            data-testid="banner-image"
            style="background-image: url(/design.jpg);"
            title="Banner Image"
          >
            <div
              class="MuiBox-root MuiBox-root-12 makeStyles-description-5"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle2"
                data-testid="banner-title"
              >
                title
              </h6>
              <p
                class="MuiTypography-root makeStyles-overflow-6 MuiTypography-caption"
                data-testid="banner-text"
              >
                Learning path description ... Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
              </p>
            </div>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-card-3 MuiPaper-elevation1 MuiPaper-rounded"
        data-testid="banner"
      >
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardMedia-root makeStyles-media-4"
            data-testid="banner-image"
            style="background-image: url(/design.jpg);"
            title="Banner Image"
          >
            <div
              class="MuiBox-root MuiBox-root-13 makeStyles-description-5"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle2"
                data-testid="banner-title"
              >
                title
              </h6>
              <p
                class="MuiTypography-root makeStyles-overflow-6 MuiTypography-caption"
                data-testid="banner-text"
              >
                Learning path description ... Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
              </p>
            </div>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
